admin/environment-management/cvms.md

```diff
@@ -21,6 +21,38 @@ supported.
 - The cluster must allow privileged containers and `hostPath` mounts. Read more
 about why this is still secure [here](#security).
 
+### Google Cloud Platform (GKE)
```

Comment:
Bit weird to abbreviate this with a different acronym...

Google Cloud Platform = GCP
Google Kubernetes Engine = GKE

Reply:
ok, maybe it should be `Google Cloud Platform w/ GKE`